plugins/Oculars/src/Oculars.cpp

```diff
@@ -1974,13 +1974,14 @@ void Oculars::paintText(const StelCore* core)
 
 	// Get the X & Y positions, and the line height
 	painter.setFont(font);
-	QString widthString = "MMMMMMMMMMMMMMMMMMMMM";
+	QString widthString = "MMMMMMMMMMMMMMMMMMMMMM";
 	const double insetFromRHS = painter.getFontMetrics().boundingRect(widthString).width();
 	StelProjector::StelProjectorParams projectorParams = core->getCurrentStelProjectorParams();
 	int yPositionOffset = qRound(projectorParams.viewportXywh[3]*projectorParams.viewportCenterOffset[1]);
-	int xPosition = qRound(projectorParams.devicePixelsPerPixel*projectorParams.viewportXywh[2] - insetFromRHS);
-	int yPosition = qRound(projectorParams.devicePixelsPerPixel*projectorParams.viewportXywh[3] - yPositionOffset - 20);
-	const int lineHeight = painter.getFontMetrics().height();
+	const double ppx = core->getCurrentStelProjectorParams().devicePixelsPerPixel;
```

Comment:
`const double ppx = core->getCurrentStelProjectorParams().devicePixelsPerPixel;` is equals to `const double ppx = projectorParams.devicePixelsPerPixel;` and now you can use `ppx` instead of `projectorParams.devicePixelsPerPixel`

Reply:
Ah, Thanks!